src/components/Alert/README.md

```diff
@@ -0,0 +1,234 @@
+
+<!--LANDING_BLOCK
+<ExampleBlock
+    code={`
+<Alert title="Rounded" message="Rounded corners" corners="rounded"  />
+<Alert title="Square" message="Square corners" corners="square" />
+`}
+>
+    <UIKit.Alert title="Rounded" message="Rounded corners" corners="rounded"  />
+    <UIKit.Alert title="Square" message="Square corners" corners="square" />
+</ExampleBlock>
+LANDING_BLOCK-->
+
+<!--GITHUB_BLOCK-->
+
+```tsx
+<Alert title="Rounded" message="Rounded corners" corners="rounded"/>
+<Alert title="Square" message="Square corners" corners="square"/>
+```
+
+<!--/GITHUB_BLOCK-->
+
+## Alert title
+
+`title` - title of an alert. Override if by Alert.Title component.
+
+<!--LANDING_BLOCK
+<ExampleBlock
+    code={`
+<Alert title={<Alert.Title className={'some-class'} text="some text"></Alert.Title>} />
+`}
+>
+    <UIKit.Alert title={<UIKit.Alert.Title className={'some-class'} text="some text"></UIKit.Alert.Title>} />
+</ExampleBlock>
+LANDING_BLOCK-->
+
+<!--GITHUB_BLOCK-->
+
+```tsx
+<Alert title={<Alert.Title className={'some-class'} text="some text"></Alert.Title>} />
+```
+
+<!--/GITHUB_BLOCK-->
+
+## Alert message
+
+`message` - The message of an alert should completely explain the alert's content.
+
+## Alert onClose
+
+`onClose` - enabled close button and defined function in click to this button.
+
+<!--LANDING_BLOCK
+<ExampleBlock
+    code={`
+<Alert onClose={() => alert('Close button pressed')} title="Alert has close" message="Alert has close" />
+`}
+>
+    <UIKit.Alert onClose={() => alert('Close button pressed')} title="Alert has close" message="Alert has close" />
+</ExampleBlock>
+LANDING_BLOCK-->
+
+<!--GITHUB_BLOCK-->
+
+```tsx
+<Alert
+  onClose={() => alert('Close button pressed')}
+  title="Alert has close"
+  message="Alert has close"
+/>
+```
+
+<!--/GITHUB_BLOCK-->
+
+### Align
+
+`baseline` - align used bu default.
+
+`center` - centered all content in vertical direction, useful if for some reason your actions take more space than text
+or
+needed icon to be in the middle of the card.
+
+<!--LANDING_BLOCK
+<ExampleBlock
+    code={`
+<Alert align="baseline" theme="info" title="Baseline" message="Baseline align" actions={<Button>button</Button>} />
+<Alert align="center" theme="info" title="Center" message="Center align" actions={<Button>button</Button>} align="center"/>
+`}>
+    <UIKit.Alert align="baseline" theme="info" title="Baseline" message="Baseline align" actions={<UIKit.Button>button</UIKit.Button>} />
+    <UIKit.Alert align="center" theme="info" title="Center" message="Center align" actions={<UIKit.Button>button</UIKit.Button>} align="center"/>
+</ExampleBlock>
+LANDING_BLOCK-->
+
+<!--GITHUB_BLOCK-->
+
+```tsx
+<Alert align="vertical" title="Vertical" message="Vertical direction" actions={<Button>button</Button>}/>
+<Alert align="horizontal" title="Horizontal" message="Horizontal direction" actions={<Button>button</Button>}/>
+```
+
+<!--/GITHUB_BLOCK-->
+
+## Properties
```

Comment:
Let's add column `Type` like [here](https://github.com/gravity-ui/uikit/tree/main/src/components/controls/TextInput#properties) and remove bold style from names

Reply:
Ok but be aware that other components have bold and different names of columns. I.e. [Button](https://github.com/gravity-ui/uikit/blob/main/src/components/Button/README.md#properties)